Project/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs

```diff
@@ -147,9 +194,18 @@ public override void Heuristic(float[] actionsOut)
     /// </summary>
     void OnCollisionEnter(Collision c)
     {
-        var force = 2000f * m_KickPower;
+        var force = m_Power * m_KickPower;
+        if (position == Position.Goalie)
+        {
+            force = m_Power;
+        }
         if (c.gameObject.CompareTag("ball"))
         {
+            // Generic gets curriculum
```

Comment:
What does this comment mean ?

Reply:
That only the generic agent gets curriculum. I can remove the if statement because the default value of m_BallTouch will be zero since it's set using curriculum.